templates/core.html

```diff
@@ -1,8 +1,20 @@
 {% extends "_base.html" %} {% block body %}
 <div class="container">
   <div class="full-width-bg component">
-    <p>Under construction. Want to help out? Visit:
-      <a href="https://github.com/ansible/jinja-docsite" target="_blank">Ansible community docsite prototype</a>
+    <p>
+      The Ansible community documentation site has recently changed.
+      The content on this page has been removed.
+    </p>
+    <p>
+      Here are some links you might find useful instead:
+      <ul>
+        <li>
+          <a href="{{ pages.site.index }}">Ansible community documentation home page</a>
+        </li>
+        <li>
+          <a href="{{ links.core_docs.collections_index }}">Indexes of all modules and plugins</a>
+        </li>
+      </ul>
     </p>
```

Comment:
This one should link to core docs. That said, I'm not sure I agree with stubbing it out. The current core page has the core link as well as the japanese  translation link. We'll always need homes for those, don't you think>?

Reply:
ok realized on another PR  that collections and core are now on the ecosystem page. So these stubs make more sense to me now. I'd say:
1 - make sure the links are correct per my comments above
2 - consider making the core in the top nav go directly to core devel

Reply:
1. done. 2. good call.

Reply:
@samccann I'm of two minds about the stub page for core.html now. I've pushed changes that link directly from the nav bar to the core docs. However that goes against the pattern of nav link to docsite page with the other nav items.

Maybe we can go ahead with this PR as-is and revisit core.html when we incorporate the translated docs?

Let me know what you think.

Reply:
sounds like a plan stan